src/server/cluster/cluster_family.cc

```diff
@@ -724,6 +727,14 @@ void ClusterFamily::MigrationConf(CmdArgList args, ConnectionContext* cntx) {
   return;
 }
 
+void ClusterFamily::Flow(CmdArgList args, ConnectionContext* cntx) {
+  CmdArgParser parser{args};
+  auto shard_id = parser.Next<size_t>();
+  DCHECK(!parser.Error());
+
```

Comment:
call thread migrate and send Ok response

Reply:
What do you mean "call thread migrate"

Reply:
cntx->conn()->Migrate(shard_set->pool()->at(flow_id));

Reply:
this connection will start replicating the data of specific shard therefore it should run on this shard

Reply:
But to migrate data we need to write serialization for it